server/statsPageRedis.js

```diff
@@ -75,15 +75,37 @@ module.exports = function(app, env) {
     try {
       const stats = await getStats()
 
+      const tableOfContents = Object.keys(stats)
+        .map(
+          (subject, i) => `
+          <b>${subject}:</b>
+          ${Object.keys(stats[subject])
+    .map((period, j) =>
+      j % 3 === 0
+        ? `
+            <ul>
+                <a href="#${i * 10 + j}">${isAdaAmountKey(period) ? `${period} (ADA)` : period}</a>
+          </ul>
+        `
+        : ''
+    )
+    .join('')}
+      `
+        )
+        .join(' ')
+
       const statsHtml = Object.keys(stats)
         .map(
-          (subject) => `
+          (subject, i) => `
             <b>${subject}:</b>
             ${Object.keys(stats[subject])
     .map(
-      (period) => `
+      (period, j) => `
               <ul>
-                <li>${isAdaAmountKey(period) ? `${period} (ADA)` : period} </li>
+                ${j % 3 === 0 ? '<a href="#tableOfContents">Table</a>' : ''}
```

Comment:
This seems weird - why don't we use as the anchor ids directly the `<subject>:<period>` key instead of numbers?